### ISSUE TITLE
Settings -> MyAccount: If I remove emails, save button is disabled

### ISSUE THREAD

**Issue by janvojt** [mff-uk/odcs#821]
Save button is enabled only after I edit text fields.

**Comment by tomas-knap** [comment]
So this is solved?

**Comment by janvojt** [comment]
It is a duplicate of #816, I did not know about #816 when I was opening this issue.
